diff --git a/node_modules/@subwallet/react-ui/es/number/index.js b/node_modules/@subwallet/react-ui/es/number/index.js
index 895129b..0081551 100644
--- a/node_modules/@subwallet/react-ui/es/number/index.js
+++ b/node_modules/@subwallet/react-ui/es/number/index.js
@@ -2,7 +2,7 @@ import classNames from 'classnames';
 import React, { useMemo } from 'react';
 import { ConfigContext } from '../config-provider';
 import useStyle from './style';
-import { balanceFormatter, formatNumber } from '../_util/number';
+import { toBNString, balanceFormatter, formatNumber } from '../_util/number';
 import Typography from '../typography';
 import { useToken } from '../theme/internal';
 const intToLocaleString = (str, separator) => str.replace(/\B(?=(\d{3})+(?!\d))/g, separator);
@@ -99,7 +99,8 @@ const Number = props => {
   }, [formatted]);
   const hideContent = useMemo(() => new Array(showHideLength).fill('*').join(''), [showHideLength]);
   return wrapSSR( /*#__PURE__*/React.createElement("div", {
-    className: classNames(classNameExtend)
+    className: classNames(classNameExtend),
+    'attr-value': toBNString(value, decimal)
   }, hide && /*#__PURE__*/React.createElement(Typography.Text, {
     className: classNames(`${prefixCls}-hide-content`),
     style: Object.assign(Object.assign({}, intStyle), {
diff --git a/node_modules/@subwallet/react-ui/es/theme/themes/dark/colors.js b/node_modules/@subwallet/react-ui/es/theme/themes/dark/colors.js
index 1e4a5c2..8e4cf50 100644
--- a/node_modules/@subwallet/react-ui/es/theme/themes/dark/colors.js
+++ b/node_modules/@subwallet/react-ui/es/theme/themes/dark/colors.js
@@ -43,9 +43,9 @@ export const generateNeutralColorPalettes = (bgBaseColor, textBaseColor) => {
     colorBgSpotlight: getSolidColor(colorBgBase, 26),
     colorBorder: getSolidColor(colorBgBase, 26),
     colorBorderSecondary: getSolidColor(colorBgBase, 19),
-    colorBgDefault: '#0C0C0C',
-    colorBgSecondary: '#1A1A1A',
-    colorBgInput: '#252525',
+    colorBgDefault: '#121317',
+    colorBgSecondary: '#1B1D24',
+    colorBgInput: '#2A2E3C',
     colorBgBorder: '#212121',
     colorBgDivider: 'rgba(33, 33, 33, 0.8)',
     colorTextLight1: colorLightBase,
diff --git a/node_modules/@subwallet/react-ui/es/theme/themes/dark/index.js b/node_modules/@subwallet/react-ui/es/theme/themes/dark/index.js
index d4a335a..250b9c5 100644
--- a/node_modules/@subwallet/react-ui/es/theme/themes/dark/index.js
+++ b/node_modules/@subwallet/react-ui/es/theme/themes/dark/index.js
@@ -1,4 +1,5 @@
 import { generate } from '@ant-design/colors';
+import { PresetColors , ThemePresetColors } from '../../interface/presetColors';
 import { defaultPresetColors, themePresetColors } from '../seed';
 import genColorMapToken from '../shared/genColorMapToken';
 import { generateColorPalettes, generateNeutralColorPalettes } from './colors';
@@ -9,7 +10,7 @@ const derivative = (token, mapToken) => {
       theme: 'dark',
       backgroundColor: token.colorBgBase
     });
-    if (colorKey === 'colorPrimary') {
+    if (ThemePresetColors.includes(colorKey)) {
       colors[5] = token[colorKey];
     }
     const colorList = new Array(10).fill(1).reduce((prev, _, i) => {
@@ -26,7 +27,7 @@ const derivative = (token, mapToken) => {
       theme: 'dark',
       backgroundColor: token.colorBgBase
     });
-    if (colorKey === 'geekblue') {
+    if (PresetColors.includes(colorKey)) {
       colors[5] = token[colorKey];
     }
     return new Array(10).fill(1).reduce((prev, _, i) => {
diff --git a/node_modules/@subwallet/react-ui/es/theme/themes/shared/genColorMapToken.js b/node_modules/@subwallet/react-ui/es/theme/themes/shared/genColorMapToken.js
index 54d952c..2653a14 100644
--- a/node_modules/@subwallet/react-ui/es/theme/themes/shared/genColorMapToken.js
+++ b/node_modules/@subwallet/react-ui/es/theme/themes/shared/genColorMapToken.js
@@ -17,10 +17,15 @@ export default function genColorMapToken(seed, _ref) {
   const primaryColors = generateColorPalettes(colorPrimaryBase);
   primaryColors[6] = colorPrimaryBase;
   const secondaryColors = generateColorPalettes(colorSecondaryBase);
+  secondaryColors[6] = colorSecondaryBase;
   const successColors = generateColorPalettes(colorSuccessBase);
+  successColors[6] = colorSuccessBase;
   const warningColors = generateColorPalettes(colorWarningBase);
+  warningColors[6] = colorWarningBase;
   const errorColors = generateColorPalettes(colorErrorBase);
+  errorColors[6] = colorErrorBase;
   const infoColors = generateColorPalettes(colorInfoBase);
+  infoColors[6] = colorInfoBase;
   const neutralColors = generateNeutralColorPalettes(colorBgBase, colorTextBase);
   return Object.assign(Object.assign({}, neutralColors), {
     colorPrimaryBg: primaryColors[1],
